src/runtime/components/elements/Feed.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <ol :class="ui.base" v-bind="attrs" :style="ui.style">
+    <template v-for="(item, key) in items" :key="key">
+      <li :class="ui.item">
+        <div>
+          <slot name="icon" v-bind="{ item, key }">
+            <div v-if="hasIcons" :class="getIconClass(item)">
+              <UIcon v-if="item.icon" :name="item.icon" />
+              <template v-else>
+                {{ key + 1 }}
+              </template>
+            </div>
+            <div v-else :class="getIconClass(item)" />
+          </slot>
+        </div>
+
+        <div>
+          <slot name="title" v-bind="{ item, key }">
+            <div :class="[ui.title.base, ui.title.align, ui.title.size, ui.title.color]">
+              {{ item.title }}
+            </div>
+          </slot>
+        </div>
+
+        <div>
+          <slot name="trailing" v-bind="{ item, key }">
+            <div :class="[ui.trailing.base, ui.trailing.align, ui.trailing.size, ui.trailing.color]">
+              {{ item.trailing }}
+            </div>
+          </slot>
+        </div>
+
+        <div>
+          <div v-if="hasSeparators && key < items.length - 1" :class="ui.separator.wrapper">
+            <div :class="ui.separator.container">
+              <div :class="[ui.separator.size, ui.separator.background, hasIcons ? '' : ui.separator.withoutIndicators]" />
+            </div>
+          </div>
+        </div>
+
+        <div :class="key < items.length - 1 ? ui.padding : ''">
+          <slot name="description" v-bind="{ item, key }">
+            <div :class="[ui.description.base, ui.description.align, ui.description.size, ui.description.color]">
+              {{ item.description }}
+            </div>
+          </slot>
+        </div>
+
+        <div />
+      </li>
+    </template>
+  </ol>
+</template>
+
+<script lang="ts">
+import { computed, toRef, defineComponent } from 'vue'
+import type { PropType, ComputedRef } from 'vue'
+import { twJoin } from 'tailwind-merge'
+import type { Strategy, FeedItem, FeedIndicatorColor } from '../../types'
+import UIcon from './Icon.vue'
+// @ts-expect-error
+import appConfig from '#build/app.config'
+import { mergeConfig } from '#ui/utils'
+import { feed } from '#ui/ui.config'
+import { useUI } from '#ui/composables/useUI'
+
+const config = mergeConfig<typeof feed>(appConfig.ui.strategy, appConfig.ui.feed, feed)
+
+export default defineComponent({
+  components: {
+    UIcon
+  },
+  inheritAttrs: false,
+  props: {
+    feed: {
+      type: Array as PropType<String[] | FeedItem[]>,
+      default: () => []
+    },
+    icons: {
```

Comment:
Do we really need this prop? It should be the default behaviour when the item has no icon and we can drop the `{{ key + 1 }}` I guess.